zkfacade/src/test/java/com/yahoo/vespa/zookeeper/CuratorTest.java

```diff
@@ -74,6 +75,18 @@ public void require_that_server_count_is_correct() {
         }
     }
 
+    @Test
+    public void require_that_server_order_is_correct() {
+        ConfigserverConfig.Builder builder = new ConfigserverConfig.Builder();
+        builder.zookeeperserver(createZKBuilder("localhost", port1));
+        builder.zookeeperserver(createZKBuilder(HostName.getLocalhost(), port2));
```

Comment:
Use a longer list that 2 to ensure it is not testing that it is reversed.

Reply:
fixed